### ISSUE TITLE
Un-minified bundled js

### ISSUE THREAD

**Issue by mmkal** [netlify/netlify-cms#6104]
**Is your feature request related to a problem? Please describe.**

Often I find myself looking at call stacks like this:

![image](https://user-images.githubusercontent.com/15040698/148462093-67aea7e3-5136-4876-94c0-97e2b187ffe6.png)

And it's usually because I've done something stupid, but netlify-cms is not being very helpful in my attempt to figure out _what_ I did.

**Describe the solution you'd like**

Two `.js` outputs in the dist folder of the `netlify-cms` package (ideally, also the various other packages like `netlify-cms-app` etc. but `netlify-cms` would be a good start). Similar to [js-yaml](https://unpkg.com/browse/js-yaml@3.12.0/dist/):

![image](https://user-images.githubusercontent.com/15040698/148462347-b6202267-6ad6-4edf-a613-2adfbc70f58c.png)

Then the docs could be updated to refer to `index.min.js` instead of `index.js`. Or, to avoid accidentally expanding bundle sizes, they could be named `index.js` and `index.unminified.js`.

**Comment by erezrokah** [comment]
Hi @mmkal, we publish the source maps so that should be good enough?

See https://unpkg.com/netlify-cms@2.10.183/dist/netlify-cms.js.map

**Comment by mmkal** [comment]
@erezrokah interesting, maybe I have chrome misconfigured. I'll look into making it load the source maps.

Although, even with source maps, it still can't be as convenient, or as well supported by all tools (chrome dev tools, Firefox dev tools, vscode, plaintext editors, vim via ssh on some server somewhere). I don't really see the downside to publishing both versions. Most libraries do it 🙃

**Comment by erezrokah** [comment]
Yes, I agree. Would you like to submit a PR for it?
I'd say use `index.unminified.js` for it not to be a breaking change

**Comment by mmkal** [comment]
@erezrokah see the commit above for a first pass. Does that look like it's roughly right?

**Comment by erezrokah** [comment]
Won't https://github.com/mmkal/netlify-cms/commit/45663be72f33574c7f38c0205aa5f92d6f2a02e0#diff-a393479db8dc8b6868eef8f5fa99c576654d75fb00336dc82c2be71789c6d4dfR86

create `<package-name>.bundle.js` and `<package-name>.bundle.unminified.js`?
I can't be sure without running it locally, though I think we'd prefer to not change the current minified name

**Comment by mmkal** [comment]
Yes you are right, just an oversight on my part. I'll fix and open a draft PR, it'll be easier to talk there.
